src/servers/graphql-old-server.ts

```diff
@@ -59,7 +79,16 @@ const resolvers = {
         id: "BTC",
         currency: "BTC",
         balance: async () => (await wallet.getBalances())["BTC"],
-        transactions: () => wallet.getTransactions(),
+        transactions: async () => {
+          const [txs, error] = await Wallets.getTransactionsForWalletId({
+            walletId: wallet.user.id,
+          })
+          if (error instanceof Error) {
```

Comment:
so if we have problem with onchain incoming transactions will we give an error to the user? (we already have transactions from ledger)

Reply:
yes we should then return both the transactions and the error via graphql... handling of error in gql layer is out of scope for this PR